### ISSUE TITLE
Broken Image link in Hardware

### ISSUE THREAD

**Issue by jweston2112** [home-assistant/frontend#13040]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

a Broken image link in settings / Hardware 

![image](https://user-images.githubusercontent.com/70152549/176536920-dfc7c936-ec1b-4f96-8cc7-e42588993649.png)

### Describe the behavior you expected

See an Icon / Image

### Steps to reproduce the issue

1. goto settings / hardware 
2.
3.
...

### What version of Home Assistant Core has the issue?

2022.7.b0

### What was the last working version of Home Assistant Core?

2022.6.7

### In which browser are you experiencing the issue with?

Google Chrome , Firefox and MS Edge

### Which operating system are you using to run this browser?

Kali Linux and Windows 10

**Comment by zsarnett** [comment]
Is it possible to have you inspect the HTML there and see what URL its requesting? per the integration it should be there..

Example of the HTML to inspect: 
![CleanShot 2022-06-29 at 16 14 51](https://user-images.githubusercontent.com/18730868/176546144-09a820ac-a1ad-4780-a10e-f352ad7c6f2f.png)

**Comment by jweston2112** [comment]
is this what you need ??

<img slot="graphic" src="https://brands.home-assistant.io/hardware/boards/dark_raspberry_pi_4.png">

**Comment by jweston2112** [comment]
I just loaded beta 4 and the Image link in Settings/System/Hardware is still a Broken Link (In Chrome and Firefox and MS Edge)

Is it waiting on another merge??

**Comment by zsarnett** [comment]
Can you make sure to clear your cache?

**Comment by zsarnett** [comment]
@frenck found that we had an incorrect file path uploaded to the brands repo. This will be fixed shortly with: https://github.com/home-assistant/brands/pull/3511

Thanks for letting us know!

**Comment by jweston2112** [comment]
Looks Good in Beta 5 ...

**Comment by osyrixx** [comment]
What about PI 3 logo?
It`s not loading, because
`GET https://brands.home-assistant.io/hardware/boards/raspberry_pi_3.png`
results in 404.
Relevant HTML is:
`<img slot="graphic" src="https://brands.home-assistant.io/hardware/boards/raspberry_pi_3.png">`

Would be great to add this also. It`s still broken :(.


<img width="619" alt="image" src="https://user-images.githubusercontent.com/14962205/179187095-78a73c2d-f272-43b7-916c-c6f174c39a8d.png">

**Comment by zsarnett** [comment]
Can you create a new issue for PI 3 specific?

**Comment by scop** [comment]
https://github.com/home-assistant/frontend/issues/13204

**Comment by iosoft** [comment]
Yes, I also face the same issue since few weeks!
**It is NOT yet fixed!**
